csrc/utils.h

```diff
@@ -469,6 +469,12 @@ class DebugPrintScope {
 
 #define DEBUG_PRINT_SCOPE(...) DEBUG_PRINT_SCOPE_NAME(__func__, ##__VA_ARGS__)
 
+#define DEBUG_LOG(...)                                    \
+  if (_debug_print_scope) {                               \
+    debug() << "[" << __FILE__ << ":" << __LINE__ << "] " \
+            << to_str("", ##__VA_ARGS__) << std::endl;    \
```

Comment:
Can not use `to_str(##__VA_ARGS__)` because, for example, if doing `DEBUG_LOG(x, y)`, the the compiler will try to create a token `to_str(x, y` which is not a valid token by C++ syntax, causing an error.